src/lib.rs

```diff
@@ -464,6 +464,12 @@ pub struct PhysicsSchedule;
 #[derive(Debug, Hash, PartialEq, Eq, Clone, ScheduleLabel)]
 pub struct SubstepSchedule;
 
+/// The schedule that runs in [`SubstepSet::PostProcessCollisions`].
+///
+/// Empty by default.
+#[derive(Debug, Hash, PartialEq, Eq, Clone, ScheduleLabel)]
+pub struct PostProcessCollisionsSchedule;
```

Comment:
`PostProcessCollisionsSchedule` feels a bit verbose and hard to read due to the length in my opinion.

For clarity and to avoid naming conflicts, we currently have e.g. `PhysicsSchedule` and `SubstepSchedule`, but maybe in this case it could be just `PostProcessCollisions` (without the `Schedule`)? Bevy doesn't use `Schedule` in its own names like `Update` and even `Main`, so I think it'd be better to leave it out for brevity.

```suggestion
pub struct PostProcessCollisions;
```

Alternatively, this could be `CollisionHooks` or `CollisionHookSchedule`, but I'm not sure if these types of systems count as hooks necessarily, so I do like the `PostProcess` name since it perhaps makes less assumptions.

Reply:
Agree with you. I was following suit with the other schedules, but agree that it's too verbose as `PostProcessCollisionsSchedule`.

One thing I am slightly concerned about is that in #150 we are discussing filtering during the broad phase as well.
I wonder if we would end up wanting a similar name in there, too?

E.g. maybe `PostProcessNarrowPhase` is more appropriate when we consider potentially wanting `PostProcessBroadPhase` in the future too?

Reply:
For the broad phase filtering, I meant just checking if `excluded_entities.contains(&entity)` in the existing systems just like how we filter by collision layers. I don't think we necessarily need any separate filtering step in the broad phase.

I'm also not entirely sure if the separate component I suggested for excluding entities is even necessary, since you could just use collision layers (or in the future maybe custom predicates) to have the same effect.

In my opinion, the only other place where a filtering step might make sense is the solver, i.e. penetration constraints. They contain the actual contact forces, and it could be useful to e.g. skip contacts when some force threshold is exceeded. However I think this would require actual hooks/callbacks, because the constraints are currently applied as soon as they're created, so a separate step wouldn't really work.

So I would just name it as `PostProcessCollisions` for now, since I find it unlikely that we add other separate filtering steps anytime soon, and if we do, we can just rename it again.

Reply:
Ok that makes sense 👍 Thanks for such a quick reply!